README.md

```diff
@@ -21,8 +21,15 @@ Or install it yourself as:
     $ gem install change_the_subject
 
 ## Usage
+The main api of the gem is the `.fix` method, which takes an array of subject terms, compares them to the configuration (the default is in `config/change_the_subject.yml`), and replaces the configured terms. You can see how Bibdata uses the gem in its [traject_config.rb](https://github.com/pulibrary/bibdata/blob/main/marc_to_solr/lib/traject_config.rb).
```

Comment:
Suggestion: Add the object that provides `.fix` -- "The main api of the gem is the `ChangeTheSubject.fix` method"

Reply:
Agreed - updated!